impeller/display_list/display_list_dispatcher.cc

```diff
@@ -577,9 +577,29 @@ static std::optional<Paint::ImageFilterProc> ToImageFilterProc(
       };
       break;
     }
+    case flutter::DlImageFilterType::kComposeFilter: {
+      auto compose = filter->asCompose();
+      FML_DCHECK(compose);
+      auto outer = compose->outer();
+      auto inner = compose->inner();
+      auto outer_proc = ToImageFilterProc(outer.get());
+      auto inner_proc = ToImageFilterProc(inner.get());
+      if (!outer_proc.has_value()) {
+        return inner_proc;
+      }
+      if (!inner_proc.has_value()) {
+        return outer_proc;
+      }
+      return [outer_proc, inner_proc](FilterInput::Ref input,
+                                      const Matrix& effect_transform) {
+        FML_DCHECK(outer_proc.has_value() && inner_proc.has_value());
+        return FilterContents::MakeComposeImageFilter(
+            input, outer_proc.value(), inner_proc.value(), effect_transform);
+      };
+      break;
```

Comment:
Please add a test that chains filters together in `impeller/display_list/display_list_unittests.cc`.

Reply:
Done.